README.md

```diff
@@ -785,6 +785,7 @@ Table of Contents
    * [getinsights.io](https://getinsights.io) - Privacy-focused, cookie free analytics, free for up to 5k events/month.
 
 ## Visitor Session Recording
+   * [FullStory.com](https://www.fullstory.com) - 1,000 sessions/month with 1 month data retention and 3 user seats. More information [here](https://help.fullstory.com/hc/en-us/articles/360020623354-FullStory-Free-Edition).
```

Comment:
please retain blank line :)

Reply:
Check it out now. Is this what you're looking for?